addons/mail/views/discuss_category_views.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="discuss_category_view_tree" model="ir.ui.view">
+            <field name="name">discuss.category.list</field>
+            <field name="model">discuss.category</field>
+            <field name="arch" type="xml">
+                <list string="Discuss Categories" editable="bottom" sample="1">
+                    <field name="sequence" widget="handle"/>
+                    <field name="name" placeholder="e.g. Service"/>
+                    <field name="channel_ids" widget="many2many_tags" domain="[('discuss_category_id', '=', False)]"/>
+                </list>
+            </field>
+        </record>
+
+        <record id="discuss_category_view_form" model="ir.ui.view">
+            <field name="name">discuss.category.form</field>
+            <field name="model">discuss.category</field>
+            <field name="arch" type="xml">
+                <form string="Discuss Categories">
+                    <sheet>
+                        <group>
+                            <field name="name" placeholder="e.g. Service"/>
+                            <field name="channel_ids" widget="many2many_tags" domain="[('discuss_category_id', '=', False)]"/>
```

Comment:
We might want to also filter out channel_type != channel, it's confusing for the admin to see all DM and other things here. (same above)